backend/webscraper/webscraper.py

```diff
@@ -1,11 +1,16 @@
+import os
+from typing import List
 from bs4 import BeautifulSoup
 import requests as req
 import datetime
+import json
+
+__location__ = os.path.realpath(
+    os.path.join(os.getcwd(), os.path.dirname(__file__)))
```

Comment:
what is this for?

Reply:
Whops forgot to delete this from debugging earlier.

I removed it.